### PR TITLE
Add anomaly detection visualizations documentation

### PR DESCRIPTION
**Pull request by vagimeli**
### Description
Add new documentation about anomaly detection visualizations

### Issues Resolved
[#458](https://github.com/opensearch-project/alerting-dashboards-plugin/issues/458)
#796 
#2075

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: images/dashboards/add-detector.png
Comment (jackiehanyang) [inline images/dashboards/add-detector.png:1]: this screenshot is outdated. please use this one instead - https://drive.corp.amazon.com/documents/hnyng@/Screenshot%202023-06-08%20at%202.17.55%20PM.png

File: _observing-your-data/ad/dashboards-anomaly-detection.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: Integrating anomaly detection with Dashboard
+parent: Anomaly detection
+nav_order: 50
+---
+
+# Integrating anomaly detection with Dashboard
+Introduced 2.8
```
Comment (jackiehanyang) [inline _observing-your-data/ad/dashboards-anomaly-detection.md:9]: this should be updated to 2.9. We're launching with 2.9 now

File: _observing-your-data/ad/dashboards-anomaly-detection.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+You can connect data visualizations to OpenSearch datasets and then create, run, and view anomaly alarms and results from visualizations in the **Dashboard** app. With only a couple clicks you can bring together traces, metrics, and logs to make your applications and infrastructure fully observable.
+
+<insert example anomaly detection visualization>
+
+## Getting started 
+
+Before getting started, you must have:
+
+- Installed OpenSearch and OpenSearch Dashboards version 2.8 or later. See [Installing OpenSearch]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/index/).
```
Comment (jackiehanyang) [inline _observing-your-data/ad/dashboards-anomaly-detection.md:22]: same as here. We need to update the version number to 2.9

File: _observing-your-data/ad/dashboards-anomaly-detection.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Integrating anomaly detection with Dashboard
+parent: Anomaly detection
+nav_order: 50
+---
+
+# Integrating anomaly detection with Dashboard
```
Comment (hdhalter) [inline _observing-your-data/ad/dashboards-anomaly-detection.md:8]: I know it's opening a can of worms, but when I see Dashboard capitalized, I think it's supposed to be OpenSearch Dashboards. Can we say instead something like, Integrating anomaly detection in visualizations? Or Adding anomaly detection to visualizations?

File: _observing-your-data/ad/dashboards-anomaly-detection.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+- Installed Alerting and Notifications Dashboards plugins. See [Managing OpenSearch Dashboards plugins]({{site.url}}{{site.baseurl}}/install-and-configure/install-dashboards/plugins/) to get started.
+
+## General requirements for anomaly detection visualizations
+
+Anomaly detection visualizations are intended for use with time-series data (real-time or historical). The following visualization types are used frequently for time-series data: line, area, vertical bar, and Time Series Visual Builder (TSVB).
+
+## Configuring admin settings
+
+ You can only access, create, or manage alerts for resources for which you have permissions. Access to anomaly detection dashboards and visualizations is controlled by OpenSearch and OpenSearch Dashboards privileges, and you can manage the settings in **Stack Management**. Access is enabled by default and appears as a feature under **Stack Management** > **Advanced Settings** > **Visualizations**. If the setting is disabled, it does not appear under Stack Management. The setting is disabled at the cluster level through the `opensearch-dashboards.yml` file.
```
Comment (hdhalter) [inline _observing-your-data/ad/dashboards-anomaly-detection.md:29]: Didn't 'Stack Management' change to "Dashboards Management'?

File: _observing-your-data/ad/dashboards-anomaly-detection.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+## Creating anomaly detection with Dashboard
```
Comment (hdhalter) [inline _observing-your-data/ad/dashboards-anomaly-detection.md:31]:
```suggestion
## Creating anomaly detectors
```

File: _observing-your-data/ad/dashboards-anomaly-detection.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+4. From the visualization panel, choose the ellipsis icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/ellipsis-icon.png" class="inline-icon" alt="ellipsis icon"/>{:/}).
+5. From the **Options** menu, choose **Anomaly Detection** > **Add anomaly detector**.
+6. Select **Create new detector**.
+7. Input information for **Detector details** and **Model features**. Up to five model features are allowed. 
+8. To preview the visualization within the flyout, toggle the **Show visualization** button.
+9. Select **Create detector**. Once you have created a new detector, the detector is added to the visualization.  
+
+![Interface of adding a detector]({{site.url}}{{site.baseurl}}/images/dashboards/add-detector.png)
+
+## Adding associated anomaly detectors
```
Comment (hdhalter) [inline _observing-your-data/ad/dashboards-anomaly-detection.md:47]:
```suggestion
## Adding anomaly detectors to visualizations
```

**Review by hdhalter — COMMENTED** [review]
Looks good Melissa! Just some suggestions/questions about headings and UI.

File: _observing-your-data/ad/dashboards-anomaly-detection.md (hunk 3 above)
Comment (vagimeli) [reply _observing-your-data/ad/dashboards-anomaly-detection.md]: Revised.

File: _observing-your-data/ad/dashboards-anomaly-detection.md (hunk 4 above)
Comment (vagimeli) [reply _observing-your-data/ad/dashboards-anomaly-detection.md]: Yes, good catch. Revised throughout this doc and the alerting doc.

File: _observing-your-data/ad/dashboards-anomaly-detection.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+![Interface and confirmation message of associating a detector]({{site.url}}{{site.baseurl}}/images/dashboards/associated-detector.png)
+
+## Refreshing the visualization
+
+Depending on the threshold settings, the visualization refreshes automatically at the specified interval. To manually refresh the visualization, select the **Refresh** button from the Dashboard page.
+
+## Next steps
+
+- [Learn more about Dashboard]({{site.url}}{{site.baseurl}}/dashboards/dashboard/index/)
```
Comment (natebower) [inline _observing-your-data/ad/dashboards-anomaly-detection.md:65]:
```suggestion
- [Learn more about the Dashboard application]({{site.url}}{{site.baseurl}}/dashboards/dashboard/index/).
```

File: _observing-your-data/ad/dashboards-anomaly-detection.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+- [Learn more about anomaly detection]({{site.url}}{{site.baseurl}}/observing-your-data/ad/index/)
```
Comment (natebower) [inline _observing-your-data/ad/dashboards-anomaly-detection.md:66]:
```suggestion
- [Learn more about anomaly detection]({{site.url}}{{site.baseurl}}/observing-your-data/ad/index/).
```

**Review by natebower — COMMENTED** [review]
@vagimeli Please see my comments and changes and let me know if you have any questions. Thanks!

File: _observing-your-data/ad/dashboards-anomaly-detection.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+
+## General requirements for anomaly detection visualizations
+
+Anomaly detection visualizations are intended for use with time-series data (real-time or historical). The following visualization types are used frequently for time-series data: line, area, vertical bar, and Time Series Visual Builder (TSVB).
+
+## Configuring admin settings
+
+Users can only access, create, or manage alerts for resources for which they have permissions. Access to anomaly detection dashboards and visualizations is controlled by OpenSearch and OpenSearch Dashboards permissions. It is enabled by default and appears as a feature under **Dashboards Management** > **Advanced Settings** > **Visualization**. If the setting is disabled, it does not appear under Dashboard Management. You can disable the setting at the cluster level through the `opensearch-dashboards.yml` file.
+
+## Creating anomaly detectors
```
[the remark this answers is not in the document]
Comment (vagimeli) [reply _observing-your-data/ad/dashboards-anomaly-detection.md:31]: Revised alerting to follow same format

File: _observing-your-data/ad/dashboards-anomaly-detection.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+## Getting started 
+
+Before getting started, you must have:
+
+- Installed OpenSearch and OpenSearch Dashboards version 2.9 or later. See [Installing OpenSearch]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/index/).
+- Installed Alerting and Notifications Dashboards plugins. See [Managing OpenSearch Dashboards plugins]({{site.url}}{{site.baseurl}}/install-and-configure/install-dashboards/plugins/) to get started.
+
+## General requirements for anomaly detection visualizations
+
+Anomaly detection visualizations are intended for use with time-series data (real-time or historical). The following visualization types are used frequently for time-series data: line, area, vertical bar, and Time Series Visual Builder (TSVB).
```
[the remark this answers is not in the document]
Comment (vagimeli) [reply _observing-your-data/ad/dashboards-anomaly-detection.md]: Revised

File: _observing-your-data/ad/dashboards-anomaly-detection.md [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+
+Use a single interface to add, view, and edit anomaly detectors that you want to associate with a visualization. Continuing with the visualization and dashboard in the preceding tutorial, follow these steps to associate an anomaly detector with a visualization:
+ 
+1. Select the ellipsis icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/ellipsis-icon.png" class="inline-icon" alt="ellipsis icon"/>{:/}) from the visualization panel, then select **Anomaly Detection**.
+2. Select **Associate a detector**.
+3. From the **Select detector to associate** dropdown menu, select the detector. Only eligible detectors are listed in the dropdown menu.
+4. View the detectors's basic information. To view comprehensive details, select **View detector page** to open the Anomaly Detection plugin page.
+5. Select **Associate detector**. An existing detector is now associated to the visualization, as shown in the following image.
+
+![Interface and confirmation message of associating a detector]({{site.url}}{{site.baseurl}}/images/dashboards/associated-detector.png)
```
Comment (vagimeli) [inline _observing-your-data/ad/dashboards-anomaly-detection.md:57]: @lezzago Will you provide me with an updated screenshot to replace this image? Thanks!

File: _observing-your-data/ad/dashboards-anomaly-detection.md [hunk 12]
```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Anomaly detection dashboards and visualizations
```
Comment (hdhalter) [inline _observing-your-data/ad/dashboards-anomaly-detection.md:3]: This still doesn't sound right to me... maybe 'Anomaly detection in dashboards and visualizations'?

File: _observing-your-data/ad/dashboards-anomaly-detection.md [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+parent: Anomaly detection
+nav_order: 50
+---
+
+# Anomaly detection dashboards and visualizations
+Introduced 2.9
+{: .label .label-purple }
+
+Anomaly detection provides an automated means of detecting harmful outliers and protects your data. When you enable anomaly detection for a metric, OpenSearch applies algorithms to continuously analyze systems and applications, determine normal baselines, and surface anomalies.
```
Comment (hdhalter) [inline _observing-your-data/ad/dashboards-anomaly-detection.md:12]:
```suggestion
OpenSearch provides an automated means of detecting harmful outliers and protecting your data when you enable anomaly detection. When applied to metrics, OpenSearch uses algorithms to continuously analyze systems and applications, determine normal baselines, and surface anomalies. 
```

File: _observing-your-data/ad/dashboards-anomaly-detection.md (hunk 12 above)
Comment (hdhalter) [reply _observing-your-data/ad/dashboards-anomaly-detection.md:3]: Do we need 'dashboards' at all? Is it assumed you have visualizations in your dashboards? Also, we don't call out adding anomaly detection to dashboards here, only visualizations.

**Review by hdhalter — COMMENTED** [review]
Looks good, Melissa, I'm just still stuck on the title. Let me know what you think. Thanks!

File: _observing-your-data/ad/dashboards-anomaly-detection.md (hunk 12 above)
Comment (vagimeli) [reply _observing-your-data/ad/dashboards-anomaly-detection.md:3]: Dashboards and visualizations are separate topics. The use case is creating dashboards that show you how your system is running. A dashboard includes visualizations and data about the anomaly details.

**Comment by vagimeli** [comment]
@hdhalter @natebower Please review changes and let me know if there is anything I didn't address that blocks this PR for merging.

**Review by hdhalter — APPROVED** [review]
Looks good!

**Pull request merged**